README.md

```diff
@@ -1,4 +1,16 @@
-# az-icons
-Can be used with Arizona Digital CDN link.
+# Arizona Icons
+
+## Using the Icons
+The icons can be used with Arizona Digital CDN link:
 
 https://cdn.digital.arizona.edu/lib/az-icons/main/az-icons-styles.css
+
+
+## Icon Creation Process
+The icons were designed using vector design software, and the font files were generated using the [IcoMoon App](https://icomoon.io/app).
```

Comment:
And icon image assets like the SVG and PNG folders?

Reply:
IcoMoon initially needs SVGs to generate the files, but it provides the *final* SVGs and PNGs along with the fonts.